src/index.tsx

```diff
@@ -158,73 +167,128 @@ export class App extends React.Component<{}, {
-                onSubmit={this.onSubmitAnswers}
-                columns={_trcGlobal._info.Columns}
-            ></AllQuestions>
+
+        let index = 0;
+
+        if (this.state.selectedRecId) {
+            index = data["RecId"].findIndex((x) => x === this.state.selectedRecId);
+        }
+
+        return (
+            <div>
+                <p>Fill in answers for: {data["RecId"][index]}</p>
+                <AllQuestions
+                    onSubmit={this.onSubmitAnswers}
+                    columns={_trcGlobal._info.Columns}
+                />
             </div>
+        )
 
     }
 
     private renderBody1() {
 
-        return <div>
-            <h2>Search</h2>
-            <p>This searches for specific individuals within the sheet and will also let you update answers for them. Use <PluginLink id={"Filter"}></PluginLink> to get counts. </p>
-            <div>Enter Search criteria: (Blanks match everything)</div>
-            <div>{_trcGlobal._info.CountRecords} total records. </div>
-            <FieldInputs Names={["RecId", "First", "Last", "Address", "City", "Zip", "PrecinctName"]} onSubmit={this.onSearch}></FieldInputs>
-
-            <h3>Search Results</h3>
-            {this.state.results && <SimpleTable data={this.state.results} ></SimpleTable>}
-            {this.state.totalFound  &&  <p>Found {this.state.totalFound} total result(s).</p>}
-            {(this.state.totalFound  && this.state.totalFound > this._topN) &&  <p>Showing first {this._topN} results. Narrow the search further.</p>}
-
-            <h3>Edit answers</h3>
-            {this.renderQuestions()}
-        </div>
+        return (
+            <PluginShell
+                description={
+                    <>
+                        <p>
+                            This searches for specific individuals within the sheet and
+                            will also let you update answers for them.
+                        </p>
+                        <p>
+                            Use <PluginLink id={"Filter"}></PluginLink> to get counts.
+                        </p>
+                    </>
+                }
+                title="Search"
+            >
+                <Panel>
+                    <Grid>
+                        <Copy>
+                            <p>Enter search criteria (blanks match everything):</p>
+                        </Copy>
+                        <Copy>
+                            <p style={{ fontWeight: 600, textAlign: 'right' }}>
+                                {_trcGlobal._info.CountRecords} total records
+                            </p>
+                        </Copy>
+                    </Grid>
+                    <FieldInputs
+                        data={_trcGlobal._contents}
+                        Names={["RecId", "First", "Last", "Address", "City", "Zip", "PrecinctName"]}
+                        Keys={["RecId", "FirstName", "LastName", "Address", "City", "Zip", "PrecinctName"]}
```

Comment:
What's the difference between names and keys?  (I see "First" vs. "FirstName") 
Can you add some comments to the control definition to aide in consumption

Reply:
We need the original keys to be able to access the data in the `data` prop. This is done to render the `datalist` for the autocomplete functionality.

I'll add documentation once with move `FieldInputs` into `trc-react`.